mpmp/src/cmds/AddMoney.java

```diff
@@ -0,0 +1,54 @@
+package cmds;
+
+import java.util.Arrays;
+
+import main.Conn;
+import main.ErrCode;
+import view.Displayer;
+
+/**
+ * add-money S->C packet
+ * @author oki
+ */
+public class AddMoney implements CmdFunc {
+	private Displayer d;
+
+	@Override
+	public void exec(String line, Conn conn) {
+		String reason;
+		String args[];
+		int amount;
+
+		args = line.split(" ");
+		if(args.length < 3) {
+			// XXX one syntax/usage error code should suffice -oki
+			conn.sendErr("Usage: add-money <amount> <reason>");
+			return;
+		}
+
+		try {
+			amount = Integer.parseInt(args[1]);
+		} catch(NumberFormatException nfe) {
+			conn.sendErr("Not a number");  // XXX make general error?
+			return;
+		}
+
+		reason = null;
+		// XXX make a static method for recombining the string -oki
```

Comment:
@leletec This is stupid; in Go, there's a `Join` method on strings; turns out that Java 8 has the same...